src/core/settings.h

```diff
@@ -65,7 +65,9 @@ typedef struct QUIC_SETTINGS_INTERNAL {
             uint64_t XdpEnabled                             : 1;
             uint64_t QTIPEnabled                            : 1;
             uint64_t RioEnabled                             : 1;
-            uint64_t RESERVED                               : 13;
+            uint64_t ResumptionTicketMinVersion             : 1;
+            uint64_t ResumptionTicketMaxVersion             : 1;
+            uint64_t RESERVED                               : 11;
```

Comment:
We shouldn't make this version configurable. I'd say remove all settings changes for this.